news/101.feature.rst

```diff
@@ -0,0 +1,5 @@
+A new reporter hook ``rejecting_candidate`` is added, replacing the former
+``backtracking`` one. The resolver triggers this hook every time it rejects
+a conflicting package before trying out an older version of it.
+The hook accepts two arguments: ``criterion`` (a ``resolvers.Criterion``
+object) and ``candidate``.
```

Comment:
```suggestion
A new reporter hook ``rejecting_candidate`` is added, replacing ``backtracking``.
The hook is called every time the resolver rejects a conflicting candidate before
trying out the next one in line.
```

The resolver does not even have a notion about _versions_; it simply goes through an iterable of candidates supplied by the provider, so the description needs to be more generic. It’s also not necessary to describe the change in detail here; that changelog is intended as a high-level overview of changes.